db/migrate/20191126181009_seed_test_results.rb

```diff
@@ -0,0 +1,63 @@
+class SeedTestResults < ActiveRecord::Migration[5.2]
+  def up
+    migrated = 0
+    ProfileHost.find_each do |profile_host|
+      host = profile_host.host
+      profile = profile_host.profile
+      test_result = TestResult.create(
+        host: host,
+        profile: profile
+      )
+
+      puts(" - Migrating results for host #{host.name}"\
+           " - profile #{profile.name}")
+      rule_results_by_start_time = scan_results(host, profile)
+        .group_by(&:start_time)
+      rule_results_by_start_time.each do |start_time, rule_results|
+        rule_result_ids = rule_results.pluck(:id)
+        puts("   - Number of RuleResults found: #{rule_result_ids.count}"\
+             " at time #{start_time}")
+        latest_rule_results = RuleResult.where(id: rule_result_ids)
+        latest_rule_results.update_all(test_result_id: test_result.id)
+        test_result.update(
```

Comment:
Let's be sure to raise errors if anything bad happens.

```suggestion
        test_result.update!(
```

Reply:
Good catch